github/search_test.go

```diff
@@ -285,16 +285,17 @@ func TestSearchService_Labels(t *testing.T) {
 		fmt.Fprint(w, `{"total_count": 4, "incomplete_results": false, "items": [{"id": 1234, "name":"bug", "description": "some text"},{"id": 4567, "name":"feature"}]}`)
 	})
 
+	repoID := int64(1234)
 	opts := &SearchOptions{Sort: "updated", Order: "desc", ListOptions: ListOptions{Page: 2, PerPage: 2}}
-	result, _, err := client.Search.Labels(context.Background(), 1234, "blah", opts)
+	result, _, err := client.Search.Labels(context.Background(), &repoID, "blah", opts)
```

Comment:
We have [`github.Int64`](https://godoc.org/github.com/google/go-github/github#Int64) helper for this, but I'm not yet sure if this change should be done.